### PR TITLE
Assess artifact dir

### PR DESCRIPTION
**Pull request by zoujas**
Removed logic for assessing artifact directory from init.mjs and placed it into a new function that is called in start.mjs.

Deleted unneeded comments from path_detect.mjs.

### CODE REVIEW

File: lib/commands/start.mjs [hunk 1]
```diff
@@ -89,11 +95,31 @@ async function uploadArtifacts(apiToken, scenario, workingDir) {
...
     }
+    
+    
+
+async function assessArtifactDirectoryValidity(artifact_dir)  {
+    let directorySize= getDirectorySize(artifact_dir);
+    let isSysDir = isSystemDirectory(artifact_dir);
+
+
+    if(directorySize>MAX_DIR_SIZE){
```
Comment (ericbeland) [inline lib/commands/start.mjs:110]: One space between each side of the operand, and one before the if, like this: ` if (directorySize > MAX_DIR_SIZE){`

File: lib/commands/start.mjs [hunk 2]
```diff
@@ -89,11 +95,31 @@ async function uploadArtifacts(apiToken, scenario, workingDir) {
...
+        log.info("WARNING. The current directory exceeds 200MB in size. Please ensure that you are using the correct directory.");
+        process.exit(1);
+    }
+   
+    if(isSysDir == true){
```
Comment (ericbeland) [inline lib/commands/start.mjs:115]: Space before if.

File: lib/commands/start.mjs [hunk 3]
```diff
@@ -152,3 +178,4 @@ async function getArtifactStatus(apiToken, artifactDir, sha) {
     }
 }
 
+}
```
Comment (ericbeland) [inline lib/commands/start.mjs:181]: Add a single carriage return at the end.

File: lib/commands/start.mjs [hunk 4]
```diff
@@ -12,17 +12,23 @@ import {decoratedError, ErrorType} from "../browserup_errors.mjs";
 import {WebConsoleClient} from "../services/webconsole_client.mjs";
 import {calculateSHA256Hash} from "../utils/hash_utils.mjs";
 import FormData from "form-data";
+import {getDirectorySize} from "../commands/detect_dir_size.mjs"
+import {isSystemDirectory} from "../commands/path_detect.mjs";
```
Comment (ericbeland) [inline lib/commands/start.mjs:16]: We should be consistent with spacing here and follow JS conventions. Look at other imports in other programs to figure out if the curly braces get a space after/before them in module imports.

**Review by ericbeland — COMMENTED** [review]
Nice work overall!!!  Left some cleanup feedback.

File: lib/commands/start.mjs (hunk 3 above)
Comment (zoujas) [reply lib/commands/start.mjs:181]: Eric, what is a single carriage return? I looked it up and the explanations did not make it obvious what you wanted. 

Did you want to eliminate the empty line in line 180, or something else?

**Review by ericbeland — APPROVED** [review]
Nice work!

**Comment by ericbeland** [comment]
Hit merge if you are ready, looks good to me!

**Pull request merged**